graph/src/runtime/asc_heap.rs

```diff
@@ -11,12 +13,14 @@ pub trait AscHeap {
     /// Allocate new space and write `bytes`, return the allocated address.
     fn raw_new(&mut self, bytes: &[u8], gas: &GasCounter) -> Result<u32, DeterministicHostError>;
 
-    fn get(
-        &self,
+    fn init<'s, 'a>(
```

Comment:
Now that this doesn't actually guarantee that the buffer is init'ed, maybe this should be renamed (back to `get` or something more creative). Also I think the `'s` lifetime can be elided.

Reply:
Done